playbook/app/pb_kits/playbook/pb_filter/docs/_filter_custom_width_react.md

```diff
@@ -0,0 +1 @@
+Filter can leverage `popoverProps` to set a custom width to the popover, without affecting the Filter size. 
```

Comment:
In thoery we can pass in any of the popover props to the filter kit using this new prop, should we call that out? Similar to [this copy](https://playbook.powerapp.cloud/kits/advanced_table/react#table-props) for tableProps?